docker/nginx/conf.d/client.conf

```diff
@@ -440,7 +440,7 @@ server {
 
 		proxy_read_timeout 600;
 		proxy_set_header User-Agent: Sia-Agent;
-		# proxy this call to siad /skynet/skylink/ endpoint (make sure the ip is correct)
+		proxy_redirect $skylink_v1 $skylink_v2;
```

Comment:
I think this warrants a comment. But more importantly, why are we fixing this in nginx as opposed to fixing it in `skyd`? I think if `skyd` is triggering the redirect, it should just be smart enough to redirect it to the requested skylink... Unless ofcourse nginx is doing the v2->v1 resolving.

Reply:
1. A comment here is a really good idea - I had to google what this does when reviewing. I should have asked for a comment.
2. It's Nginx going the resolving. Here's the discussion: https://discord.com/channels/542938080349519882/834800136092909578/862318336531759164

Reply:
Yeah it's nginx that does the resolving because we want to potentially return the resource from nginx cache. I will add a proper comment to this section.